test/job/validate.spec.ts

```diff
@@ -11,39 +11,81 @@ describe('validate', () => {
 
   beforeEach(async () => jest.clearAllMocks());
 
-  it('validates a job', () => {
-    const job: Job = toJob({ name: 'test', interval: '1 minute', handler: () => 'finished' });
+  it('validates a job with an interval', () => {
+    const job: Job = toJob({
+      name: 'test',
+      schedule: { interval: '1 minute', firstRunAfter: 0 },
+      handler: () => 'finished',
+    });
+    expect(validate(job, logger)).toBe(true);
+  });
+
+  it('validates a job with a cron schedule', () => {
+    const job: Job = toJob({
+      name: 'test',
+      schedule: { cronSchedule: '0 9 * * 1-5' },
+      handler: () => 'finished',
+    });
     expect(validate(job, logger)).toBe(true);
   });
 
   it('reports error when interval cannot be parsed', () => {
-    const job: Job = toJob({ name: 'test', interval: 'not an interval', handler: () => 'finished' });
+    const job: Job = toJob({
+      name: 'test',
+      schedule: { interval: 'not an interval', firstRunAfter: 0 },
+      handler: () => 'finished',
+    });
     expect(validate(job, logger)).toBe(false);
 
     expect(logger.error).toHaveBeenCalledTimes(1);
     expect(logger.error).toHaveBeenCalledWith(
       'job cannot be defined',
       MomoErrorType.defineJob,
-      { name: job.name, interval: job.interval },
+      { name: job.name, interval: 'not an interval' },
       momoError.nonParsableInterval
     );
   });
 
   it('reports error when interval is not positive', () => {
-    const job: Job = toJob({ name: 'test', interval: '-1 minute', handler: () => 'finished' });
+    const job: Job = toJob({
```

Comment:
maybe we can introduce two describe blocks here, one for interval, one for cron, to group the test cases a bit better